### DISCUSSION TITLE
Pro Tools  network and error cpu Ag plugin

### DISCUSSION THREAD

**Discussion by taxawow** [apohl79/audiogridder#1196] [Help]
I am using AG with pro tools and my network I/O is 700kb/s and 500kb/s, more or less, my system is apple M2, 16gb ram, ventura 13.2.1. I am using pro tools 2023.3 and connected via ethernet to a windows 10 pc and it is giving me 100% cpu picks every 10 seconds.
Then I start a session in Cubase 12 and I follow the same steps and everything works perfectly giving network 7mb /s and 6.50mb /s. and the same in logic, the only thing that fails is pro tools and I really don't understand it. Because I don't have the same network, I'm missing something, some special permission, if Cubase and logic failed me, I would understand, but only pro tools? If someone had this same problem, I would appreciate your help, it's because I need to use pro tools for some jobs that they ask me for. Thank you

**Comment by apohl79** [comment]
Every 10 seconds sounds a bit like it is spiking when the AG plugin is fetching the loaded plugin states from the server. Is this happening on all AG plugins or could you narrow it down to a particular plugin you are using?

You could try to set the "Remote Sync Frequency" in the plugin settings menu to "When an editor is active (every 10s)" and close the plugin UIs and see if that helps.
